src/component-library/CoinIcon/LPCoinIcon.tsx

```diff
@@ -0,0 +1,64 @@
+import { forwardRef, useCallback } from 'react';
+
+import { Icon } from '../Icon';
+import { CoinIconProps } from './CoinIcon';
+import { FallbackIcon } from './FallbackIcon';
+import { coins } from './utils';
+
+type Props = {
+  tickers: string[];
+};
+
+type InheritAttrs = Omit<CoinIconProps, keyof Props>;
+
+type LPCoinIconProps = Props & InheritAttrs;
+
+const LPCoinIcon = forwardRef<SVGSVGElement, LPCoinIconProps>(
+  ({ tickers, size = 'md', ticker, ...props }, ref): JSX.Element => {
+    const [tickerA, tickerB, tickerC, tickerD] = tickers;
```

Comment:
Not a blocker and something we can come back to, but could this be done by looping through the values rather than checking each one separately (i.e. we have code which explicitly checks the value of `tickerA`, `tickerB` etc.—could this component be rendered without those individual checks?

Nothing to do on this now, but something to consider next time we do some work on this component.

Reply:
I think it would make the component more complex and we do not expect more than 4 assets at the moment. I also use each asset in logic that will determine each positions.